api/v1/servicebinding_validating_webhook.go

```diff
@@ -124,3 +130,12 @@ func (sb *ServiceBinding) validateCredRotatingConfig() error {
 
 	return nil
 }
+
+func (sb *ServiceBinding) validateAnnotations() error {
+	if sb.Annotations != nil {
+		if _, ok := sb.Annotations[api.IgnoreNonTransientErrorAnnotation]; ok {
+			return fmt.Errorf("annotation %s is not suppoted in service binding", api.IgnoreNonTransientErrorAnnotation)
```

Comment:
@danielforsap review error message

Reply:
The specified annotation '%s' is not supported within the service binding object. 
@I065450  can we say that?